karaf/apache-brooklyn/src/main/resources/bin/setenv

```diff
@@ -85,6 +89,9 @@ export EXTRA_JAVA_OPTS="-Dbrooklyn.location.localhost.address=127.0.0.1 ${EXTRA_
 # Increase garbage collection, see https://issues.apache.org/jira/browse/BROOKLYN-375
 export EXTRA_JAVA_OPTS="-XX:SoftRefLRUPolicyMSPerMB=1 ${EXTRA_JAVA_OPTS}"
 
+# Set the DNS TTL for the JVM
+export EXTRA_JAVA_OPTS="-Dsun.net.inetaddr.ttl=${DNS_TTL} ${EXTRA_JAVA_OPTS}"
```

Comment:
It would be nicer not to use the `sun` private system property but I take your point about the other value being a security property.  I had a look to see if Karaf would let us specify this somehow but I don't see a way to do that.  (You can configure security providers but there doesn't seem to be an equivalent for security properties.)

At the least though I would add some comment here explaining the use of `sun.net` - even just copy the text from the description of this PR.

Reply:
@geomacy Indeed, it is annoying that we cannot control the `networkaddress.cache.ttl` directly hence why I went for the old `sun.net.inetaddr.ttl` property.

Good idea though, I'll add a comment